help/destinations/catalog/data-partner/acxiom-audience-connectors.md

```diff
@@ -0,0 +1,215 @@
+|Address Line 1      |Address 1 field of primary residence                                                                                              |5            |255         |
+|Address Line 2      |Address 2 field of primary residence                                                                                              |6            |255         |
+|City                |City of primary residence                                                                                                         |7            |255         |
+|State               |State abbreviation of primary residence                                                                                           |8            |2           |
+|Zip Code            |Full zip code of primary residence                                                                                                |9            |10          |
+|Email               |Primary email By default, this field is used as a deduplication key to make the records unique                                    |10           |255         |
+|Phone               |Phone number of individual (area code + number) By default, this field is used as a deduplication key to make the records unique  |11           |10          |
+
+
+At a minimum, the following fields are required:  Last Name, Address Line 1, City, State, Zip Code.  If other fields are not available, then the previously mentioned fields should remain blank.
+
+For additional information on mapping custom audiences in [!DNL Adobe Experience Platform], see the Mapping section in the [Data Prep UI Guide](https://experienceleague.adobe.com/en/docs/experience-platform/data-prep/ui/mapping). The Source Fields are [!DNL Adobe] standard fields that come from the [!DNL Adobe] schema. The Target Fields are the predefined fields required by [!DNL Acxiom’s Audience Connectors].
+
+>[!NOTE]
+>
+>You cannot map additional fields.
+
+If you are not using [!DNL Adobe’s] standard schema, see the [Query Service UI guide](https://experienceleague.adobe.com/en/docs/experience-platform/query/ui/overview) documentation for information on how to use the query service to populate the [!DNL Adobe] standard schema with your field names. 
+
+
+
+
+### Review {#review}
+Once you have completed all the steps above, you have an opportunity to review your destination connection status and audience details before activating (distributing) it. The audiences you selected will show up at the bottom in a list. Each audience will be a separate call to [!DNL Acxiom’s Audience Connectors] API. 
+
+If you are happy with the results, select **Finish** to activate your destination. 
+
+![Review your audience](../../assets/catalog/data-partner/acxiom/review_audience.png)
+
+
+## Activate audiences to this destination {#activate}

```

Comment:
There are multiple issues in this section.

1. I don't understand why this section is here. Please follow the headings in the template (https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template).
2. The activation-related information in this section should be higher up the page, in the activation section. I'm talking specifically about this part: "Once your audience is distributed to the requested destination, contact your destination representative to begin setting up your campaign." Please also rephrase that to "Once your audience is distributed to the selected destination, contact your destination representative to begin setting up your campaign."
3. The second point, where you address troubleshooting, is a different topic. Please create a new `## Troubleshooting` heading, and include the second part of this section there.

Reply:
Done